docs/testcases/TC06-005-pass.html

```diff
@@ -137,7 +137,7 @@ <h4 id="TCcode">Code</h4>
   								<h4 id="TCurl">Single-Page URL</h4>
```

Comment:
TC06-005-pass.html:

  - Minor issue: The code `<p>` value does not match the actual `<p>` value in the embedded test file. The difference is minor but different.

Reply:
Fixed